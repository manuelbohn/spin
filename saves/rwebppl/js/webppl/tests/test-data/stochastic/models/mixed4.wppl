var model = function() {
  var x = uniformDraw([0.5, 18]);
  var y = gaussian(1, Math.sqrt(x / 2));
  return y;
};
