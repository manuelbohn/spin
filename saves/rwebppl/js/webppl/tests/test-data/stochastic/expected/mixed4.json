{
  "mean": 1,
  "std": 2.157
}
